### ISSUE TITLE
Dark Zenhub Cards and Buttons unstyled

### ISSUE THREAD

**Issue by altern8tif** [StylishThemes/GitHub-Dark#407]
### Unstyled Content
- [x] No other existing issue and/or pull request.
- [ ] URL of unstyled content.
- [ ] If unable to provide a URL, please report the class name of unstyled content.
- [ ] Provide steps to reproduce (opening a dialog, etc).
### Feature Request

I'm new to the repo so I'm not so sure how the Dark Zenhub UI is supposed to look like.  But based on what I'm seeing, it seems like something is off?

![screen shot 2016-09-16 at 3 15 23 pm](https://cloud.githubusercontent.com/assets/5793364/18578197/2d58e3f2-7c21-11e6-875d-78e5bac0f0ae.png)

The cards and buttons are not styled in the right attributes/colours even with the `!important` tags.

**Comment by Mottie** [comment]
Hi @altern8tif!

I think you might be using an older style. Please manually update directly from userstyles.

**Comment by altern8tif** [comment]
Thanks! It works fine now.  I guess it's a Stylish issue that it doesn't update even after "applying updates"...

**Comment by Mumumurilo** [comment]
Hi,

Sorry to bring this issue up again after almost three years, but I've just installed the theme and seems like the issue is still pertinent. Maybe it is because I'm using GitHub Enterprise and had to add the enterprise URL manually?

![Screen Shot 2019-06-14 at 11 56 01](https://user-images.githubusercontent.com/8595653/59518130-68c00300-8e9b-11e9-9c40-f851add0ca89.png)

**Comment by silverwind** [comment]
ZenHub should look much better in the latest version. It almost looks like no styles do match. Does your `<body>` have a `zhio` class on it?

**Comment by silverwind** [comment]
Those classes are fine, all our ZenHub selectors are like `body.zhio .zhc-pipeline`. Please post a screenshot of the inspector where `<body>` is visible.

**Comment by Mumumurilo** [comment]
Seems like my `<body>` does not have the `zhio` class. The style works fine if I add it manually:

<img width="1280" alt="Screen Shot 2019-06-17 at 14 05 36" src="https://user-images.githubusercontent.com/8595653/59622810-02d8b300-9109-11e9-83f8-b4d34751f4f5.png">

Here's my full `<body>` tag:

`<body class="logged-in enterprise env-production emoji-size-boost zhe zh-navbar-top zh_logged_in" style="overflow: hidden;">`

**Comment by silverwind** [comment]
Thanks, we can probably fix that, will check out out later.

You may want to try https://github.com/StylishThemes/Overlay-Scrollbars to darken those scrollbars 😉

**Comment by Mumumurilo** [comment]
@silverwind Thank you! I'll check it out.

**Comment by GitHubDark** [comment]
This is what I see on Public ZenHub:

`logged-in env-production emoji-size-boost min-width-lg zhio zh-navbar-top zh_logged_in`

Going to use `.zh_logged_in`.

**Comment by Mumumurilo** [comment]
Sorry, but I've just found another issue when opening the cards. Seems like only the `timeline-comment-group` and its children are appending the style.

<img width="1280" alt="Screen Shot 2019-06-18 at 11 21 53" src="https://user-images.githubusercontent.com/8595653/59692699-e5185600-91bb-11e9-9919-523170c0d123.png">

By tweaking with the style on the developer view I was able to find an `!important` tag appended to a `background` parameter on the `partial-discussion-header` div. However, I couldn't find what is making the rest of the modal still have the white background.

<img width="1280" alt="Screen Shot 2019-06-18 at 11 32 45" src="https://user-images.githubusercontent.com/8595653/59693166-d9795f00-91bc-11e9-9f84-19ee68863ed3.png">
